src/components/document/canvas.tsx

```diff
@@ -92,9 +92,11 @@ export class CanvasComponent extends BaseComponent<IProps, IState> {
     this.showPlaybackControlsDisposer = reaction(
       () => this.props.document?.showPlaybackControls,
       () => {
-        this.setState((prevState, props) => {
-          return this.updateHistoryDocument(prevState);
-        });
+        if (this.props.showPlayback) {
```

Comment:
This was a bug introduced in the change to the playback button toolbar change.  Without this conditional test documents on the "right side" were also updating when the left side playback sliders were shown.